.pre-commit-config.yaml

```diff
@@ -12,6 +12,9 @@ repos:
   - id: check-merge-conflict
   - id: check-yaml
   - id: debug-statements
+  - id: file-contents-sorter
+    files: requirements/(base|docs|local|test)\.in
```

Comment:
Oh neat, I didn't know about this one! Looks like it would be handy in lots of places.